src/main/java/zeenea/connector/SourceItem.java

```diff
@@ -0,0 +1,403 @@
+package zeenea.connector;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.time.Instant;
+import java.util.*;
+import zeenea.connector.common.ItemIdentifier;
+import zeenea.connector.contact.SourceContactRelation;
+import zeenea.connector.property.*;
+
+/**
+ * Base class for all kinds of source items. An item is something that is managed and searchable in
+ * Zeenea Data Catalog.
+ *
+ * @since 1.0.0
+ */
+public abstract class SourceItem {
+
+  private final String name;
+
+  private final ItemIdentifier id;
+
+  private final String description;
+
+  private final Map<String, SourcePropertyValue> metadata;
```

Comment:
Proposition d'un nommage plus homogène : 
```suggestion
  private final Map<String, SourcePropertyValue> properties;
```

Reply:
Modifié